src/Components/Authorization/test/AuthorizeRouteViewTest.cs

```diff
@@ -76,6 +76,74 @@ public void WhenAuthorized_RendersPageInsideLayout()
                 edit => AssertPrependText(batch, edit, "Hello from the page with message: Hello, world!"));
         }
 
+        [Fact]
```

Comment:
The formatting on this file is a little off. Try running the formatter in VS.

If you're developing in VS Code, I've had a lot of success with the `C# FixFormat Fixed` extension.

Reply:
Think I see the formatting issues you're talking about - I moved the brace off of the Method Definition line